docs/pages/setup/helm-reference.mdx

```diff
@@ -19,3 +21,14 @@ Deploy the Teleport Kubernetes Service, Application Service, or Database Service
 
 </Tile>
 </TileSet>
+
+## Plugins
+
+<TileSet>
+<Tile href="./helm-reference/teleport-plugin-event-handler.mdx" icon="kubernetes" title="teleport-plugin-event-handler">
```

Comment:
If we're going to add more tiles to this page, I think we can remove the SVG link from each tile. This will let more tiles fit on the page, and we won't have to keep choosing SVG icons that aren't necessarily appropriate for the associated guides.

Reply:
Let's try it!

Reply:
@ptgott, what do you think about breaking the charts in two groups? It'd look like this:

<img width="795" alt="image" src="https://user-images.githubusercontent.com/514352/179510441-f1084ea5-fc84-4293-a2b6-af05c51e78e3.png">

It feels a bit confusing to I just add the new tile to the existing tileset (knowing that there'll be 4 more):

<img width="1194" alt="image" src="https://user-images.githubusercontent.com/514352/179510570-9556d9fe-51c2-4be3-a25b-7f08e97520f4.png">